numpy/core/tests/test_multiarray.py

```diff
@@ -6355,50 +6360,52 @@ def test_string(self):
         assert_equal(np.where(False, b, a), "abcd")
 
 
-class TestSizeOf(TestCase):
+if not IS_PYPY:
```

Comment:
Another way to do this in Nose is probably to `raise SkipTest` in `def setup(self): ...`